### PR TITLE
update recipe to the new format

### PR DESCRIPTION
**Pull request by ocefpaf**
@conda-forge-admin, please rerender

### CODE REVIEW

**Comment by bgruening** [comment]
@conda-forge-admin, please rerender

**Comment by PertuyF** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,50 +1,76 @@
...
   version: {{ version }}
 
 source:
-  url: https://www.ctan.org/tex-archive/systems/texlive/Source/texlive-{{ version }}-source.tar.xz
-  sha256: c0a032aa9fa29e7cb521659779d6415f3b4b062a6a44193fb4e79cfefe7b8954
+  - url: https://www.ctan.org/tex-archive/systems/texlive/Source/texlive-{{ version }}-source.tar.xz
+    sha256: fe0036d5f66708ad973cdc4e413c0bb9ee2385224481f7b0fb229700a0891e4e
+    patches:
+      - texlive-20180414-source-upstream_fixes-1.patch
+  # 2.5GB download ..
```
Comment (mariusvniekerk) [inline recipe/meta.yaml]: errr.  I dont thin that'll work

Comment (ocefpaf) [reply recipe/meta.yaml]: Nope. It does not :smile:

Comment (ocefpaf) [reply recipe/meta.yaml]: Probably. I don't have to time to investigate that now though, sorry.

**Comment by mariusvniekerk** [comment]
@conda-forge-admin please rerender

**Pull request closed without merging**